### ISSUE TITLE
ENH: New interface for Coregistration

### ISSUE THREAD

**Issue by GuillaumeFavelier** [mne-tools/mne-python#8833]
This PR summarizes the ideas to implement for `coreg`:

<details open>

<summary><b>Milestone 1.0</b></summary>

- [ ] increase fiducial picking performance (https://github.com/mne-tools/mne-python/pull/9689#issuecomment-951910957)
- [ ] add support for the `guess_mri_subject` parameter 
- [ ] add support for the `project_eeg` parameter
- [ ] add support for the `scale_by_distance` parameter
- [ ] add support for the `mark_inside` parameter
- [ ] add support for the `scale` parameter
- [ ] add support for the `head_inside` parameter
- [ ] find a better name for the "Digitization Source" section (which could be confusing)
- [ ] make the sections collapsible *(ipywidgets)*
- [ ] add units to UI (Omit more explicit message)
- [ ] console idea to add to the UI whatever is printed in the terminal
- [ ] add tooltips as much as possible on buttons
- [ ] add legend to plot alignment / coreg to see what colored points are etc.
- [ ] display dist estimation in mm.
- [ ] 📁 icon for file button widgets (instead of "Load")

</details>

<details>
<summary><b>Done</b></summary>

- [x] Update `Coregistration()` to require parameters (https://mne.discourse.group/t/mne-coreg-based-on-ctf-data/3022/4)
- [x] Decouple GUI code from model (suggested in https://github.com/mne-tools/mne-python/pull/6693#issuecomment-542364434, done in https://github.com/mne-tools/mne-python/pull/9516)
- [ ] Build a coreg GUI using `pyvista` (work in progress in https://github.com/mne-tools/mne-python/pull/9689)

</details>

<details open>
<summary><b>Bugs</b></summary>

- [ ] inconsistent behaviour of checkbox layout for ipywidgets

</details>

**Comment by hbk008** [comment]
I am not able to get past this: 

from mne.coreg import Coregistration

it throws error: cannot import name 'Coregistration' from 'mne.coreg' 

The other function from mne.coreg work well like getting fiducials from fiducials = mne.coreg.get_mni_fiducials(subject, subjects_dir=subjects_dir)
but this import Coregistration from mne.coreg fails for some reason. 
Can someone help me resolve this?

**Comment by hoechenberger** [comment]
> from mne.coreg import Coregistration

To start up the coregistration GUI, you need to call

```python
mne.gui.coregistation()
```

**Comment by hbk008** [comment]
Yes, but I want to use automated coregistration as explained in MNE tutorial: https://mne.tools/dev/auto_tutorials/forward/25_automated_coreg.html

So, how should I resolve this error?

**Comment by hoechenberger** [comment]
@hbk008 I cannot reproduce this with MNE-Python 0.24. Please update to MNE-Python 0.24, then it should work.

Please note that the issue tracker is intended for bug reports and development issue tracking only. For usage questions like yours, please refer to our forum at https://mne.discourse.group

**Comment by hbk008** [comment]
Okay, will post in the forum next time. You meant update from 0.23 to 0.24, right? Anyway, I updated to 0.24 and it worked finally, thanks!

**Comment by hoechenberger** [comment]
Great to hear it works now!! 👍
